jupyter_scheduler/handlers.py

```diff
@@ -272,7 +272,7 @@ def output_files_manager(self):
     @tornado.web.authenticated
     async def get(self, job_id):
         redownload = self.get_query_argument("redownload", False)
-        self.output_files_manager.copy_from_staging(job_id=job_id, redownload=redownload)
+        await self.output_files_manager.copy_from_staging(job_id=job_id, redownload=redownload)
```

Comment:
Does this solve the frontend issue of knowing when to show the links?

Reply:
No, this won't fix the front end issue.